types/braintree-web-drop-in/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
new definitions cannot have anything except extends rule in tslint. Please fix any errors reported by linter after removing rules section,